services/ui-src/package.json

```diff
@@ -192,9 +192,8 @@
     "@babel/plugin-transform-react-jsx": "^7.12.17",
     "core-js": "^3.9.1",
     "eslint-config-prettier": "^7.1.0",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
```

Comment:
left final eslint in devDependencies because eslint should not be in the deployed build

Reply:
same for identity-obj-proxy